docs/src/languages/php/fpm.md

```diff
@@ -1,103 +1,114 @@
 ---
 title: "PHP-FPM sizing"
+description: "Learn how to adjust the maximum number of PHP-FPM workers for your app"
 weight: 5
 ---
 
-Platform.sh uses a heuristic to automatically set the number of workers of the PHP-FPM runtime based on the memory available in the container. This heuristic is based on assumptions about the memory necessary on average to process a request. You can tweak those assumptions if your application will typically use considerably more or less memory.
+PHP-FPM helps improve your app's performance
+by maintaining pools of workers that can process PHP requests.
+This is particularly useful when your app needs to handle a high number of simultaneous requests. 
 
-Note that this value is independent of the `memory_limit` set in `php.ini`, which is the maximum amount of memory a single PHP process can use before it is automatically terminated.
-These estimates are used only for determining the number of PHP-FPM workers to start.
+By default, Platform.sh automatically sets a maximum number of PHP-FPM workers for your app. 
+This number is calculated based on three parameters:
+- The amount of memory you can allot for PHP processing depending on your server specifications.
+- The request memory.
+  This is the amount of memory an average PHP request is expected to require.
+- The reserved memory. 
+  This is the amount of memory you need to reserve for tasks that aren't related to requests. 
 
-## The heuristic
+Follow the steps to adjust the maximum number of PHP-FPM workers depending on your app's needs.
 
-The heuristic is based on three input parameters:
+## Before you begin
 
- * The memory available for the container, which depends on the size of the container (`S`, `M`, `L`),
- * The memory that an average request is expected to require,
- * The memory that should be reserved for things that aren't specific to a request (memory for `nginx`, the op-code cache, some OS page cache, etc.)
+You need:
 
-The number of workers is calculated as:
+- An up-and-running web app in PHP, complete with [PHP-FPM](https://www.php.net/manual/en/install.fpm.php)
+- The [Platform.sh CLI](../../administration/cli/_index.md)
 
-![FPM](/images/php/phpfpmworkers.png "0.3")
```

Comment:
I think a formula like this can sometimes be better than words and bring a lot of clarity. But in this case, I removed it because it actually made everything more confusing to me, and I'm still not sure I completely understand what each element in it means. Granted, I'm terrible at maths, but I figured if it made things more obscure for me, then it might very well be the case for other people too.

Reply:
I looked into it some more and the `max` and `2` bits mean that it sets a minimum value of 2 (so if the other part of the formula comes out to less than 2, the result is 2). That information is missing elsewhere, but is definitely not ideally presented in the image (let alone only being in image form).

The other three parts of the formula are the three bullet points above. So it should be possible to make them clearer. I think seeing that RequestMemory is on the bottom of the equation is helpful to seeing quickly that it's the inverse -- so increasing it decreases the number of workers.

I also discovered that the result is probably rounded up when it's a fraction. Not sure how much that info helps, but it might.

So I'd replace it with just the three items and include the minimum elsewhere in the text. At the very least, including the minimum.